picard/file.py

```diff
@@ -176,6 +177,25 @@ def _load_check(self, filename):
                 raise
             return file_format._load(filename)
 
+    @staticmethod
+    def _load_check_metadata(stopping, process_queue, result_queue):
+        from picard.formats import open_ as open_file
+        while True:
+            filename = process_queue.get()
+            if stopping.value:
+                break
+            result_queue.put((filename, File._load_check_metadata_thread(open_file, filename)))
+        result_queue.cancel_join_thread()
+
+    @staticmethod
+    def _load_check_metadata_thread(open_file, filename):
+        try:
+            file = open_file(filename)
+            result = file._load_check(filename)
+        except Exception:
+            result = None
+        return result
```

Comment:
Is it possible to reduce the scope of exception catching?
In which cases and what are exceptions possibly thrown?
Would it make sense to log the exception somehow?

Also we may want to just return the result instead of storing it in a variable.
```python
        try:
            file = open_file(filename)
            return file._load_check(filename)
        except Exception:
            return None
```

Reply:
Not really sure, it can catch a bunch of mutagen weird exceptions, filesystem ones, etc. Same as above: corrupted files, moved files, filesystem errors. 
Not sure, AFAIK Picard didn't log these errors, but forwareded it to the next step, where the error result is checked and file gets updated with the red dot indicating an error.
The last one, sure. Used to that style due to debugging.